glide.lock

```diff
@@ -22,6 +22,37 @@ imports:
   version: fa29b1d70f0beaddd4c7021607cc3c3be8ce94b8
   subpackages:
   - capnslog
+- name: github.com/docker/distribution
```

Comment:
I don't see this in use else where in this PR. Was it just not added some other time when it should have been?